commerce-cloud-code/app_yourshophere/cartridge/controllers/Order.js

```diff
@@ -0,0 +1,9 @@
+
+const server = require('server');
+
+server.get('Confirm', (req, res, next) => {
+    res.page('order-thank-you', {});
```

Comment:
Maybe pass the order no here as aspect, that would allow to leverage preview in Page Designer with real orders.

Reply:
Yes thats not done, the is just a first draft seeking feedback for some of the approaches